### PR TITLE
Fix searchbar and update parts data

### PR DESCRIPTION
**Pull request by jessodri**
Hey Mike,

This is just a pull request to fix the searchbar (you should be able to search by pretty much anything now) and add the code, category and used fields to the serviceItems data.

Please review and approve.

Thanks,
Jess

### Done Checklist

* [ ] Task has been completed as per the Trello card
* [ ] All acceptance criteria are met
* [ ] Developer has tested this locally
* [ ] Developer has reviewed their own PR
* [ ] Design has been reviewed by design team
* [ ] Tests have been added or updated
* [ ] Views are responsive for mobile devices
* [ ] Feature files have been written or updated
* [ ] Changelog updated

### CODE REVIEW

**Comment by jessodri** [comment]
Actually I see some tests are failing, I will fix those up first

File: meteor-app/imports/ui/assessment/assessment-add.js [hunk 1]
```diff
@@ -108,11 +108,14 @@ class AssessmentAdd extends Component {
...
-            price: key.price
+            price: key.price,
+            code: key.code,
+            category: key.category,
+            used: key.used
           }
         })
       const custName = formData.b2bRefurbish ? 'back2bikes' : formData.name.toLowerCase()
-      const search = custName + formData.email + formData.bikeMake.toLowerCase() + formData.bikeColor.toLowerCase()
+      const search = custName + formData.email + formData.phone + formData.bikeMake + formData.bikeModel + formData.bikeColor + formData.mechanic + formData.assessor + formData.comments + formData.package + JSON.stringify(partsItem)
```
Comment (mikkelking) [inline meteor-app/imports/ui/assessment/assessment-add.js]: This code works fine, but there is a way that we can future proof your code like this.
```
const search = Object.values(formData).join(" ")
```
So if and when (which is likely) we add more information to the form, we won't need to change this code again.

File: meteor-app/imports/startup/server/assessment-data.js [hunk 2]
```diff
@@ -80,219 +80,381 @@ Meteor.methods({
...
       {
         name: 'Cables x 4',
-        price: 2000
+        price: 2000,
+        code: 'O',
+        category: 'cable',
+        used: false
       },
       {
         name: 'Cable fitted x 1',
```
Comment (mikkelking) [inline meteor-app/imports/startup/server/assessment-data.js]: As we are introducing items for front and rear brake cables, we can get rid of these x1, x2 etc

File: meteor-app/imports/ui/assessment/assessment-job-card.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ class JobCard extends Component {
 JobCard.propTypes = {
   currentJob: PropTypes.shape({
     _id: PropTypes.string,
-    status: PropTypes.string
+    status: PropTypes.number
```
Comment (mikkelking) [inline meteor-app/imports/ui/assessment/assessment-job-card.js]: I think a job should always have a status value, so I think you should add `.isRequired` to this

File: meteor-app/imports/ui/assessment/assessment-job-card-list.js [hunk 4]
```diff
@@ -100,9 +100,9 @@ export default withTracker(props => {
 
   const renderJob = () => {
     if (statusLine == '') {
-      return Assessment.find({ search: { $regex: searchLine } }).fetch()
+      return Assessment.find({ search: { $regex: searchLine, $options: "i" } }).fetch()
```
Comment (mikkelking) [inline meteor-app/imports/ui/assessment/assessment-job-card-list.js]: I think this is a user entered string, and we are asking it to be treated as a regular expression (with case insensitivity) to find something that matches. 

On the surface that seems sensible, but just try entering `\` into the search box and see what happens.

File: meteor-app/imports/api/fake-data.js [hunk 5]
```diff
@@ -2,21 +2,31 @@ import faker from 'faker'
...
 export const fakeJob = () => {
-  // faker.seed(123)
+
+  // search items
   const name = faker.name.findName()
   const make = faker.random.arrayElement(['Apollo', 'Giant', 'Malvern Star', 'Specialized'])
   const color = faker.commerce.color()
+  const email = faker.internet.email()
+  const phone = faker.phone.phoneNumberFormat()
+  const model = faker.random.arrayElement(['TZ1000', 'Fireball', 'Chicken Dance', 'Zoom2000'])
```
Comment (mikkelking) [inline meteor-app/imports/api/fake-data.js]: Haha, a bike called a 'Chicken Dance'? Better choices would be 'Trail Breaker', 'Samurai', 'Katana', 'Yukon', 'Cypress', 'Defy', 'CX1'

**Comment by mikkelking** [comment]
I notice all the services have a total price of $7, which doesn't seem right somehow. I know it's just fake data right now, but it will feel better if it's an actual calculation of the random items added.
Also the estimated values are coming out as odd amounts with cents in. Probably a multiple of $50 in a range of $100 to $500 would be more realistic

**Comment by jessodri** [comment]
Thanks Mike, I've made some changes, please review.

**Pull request merged**